Makefile

```diff
@@ -61,6 +61,6 @@ noobaa: base
 
 clean:
 	@echo Stopping and Deleting containers
-	@docker ps -a | grep noobaa_ | awk '{print $$NF}' | xargs docker stop
-	@docker ps -a | grep noobaa_ | awk '{print $$NF}' | xargs docker rm
+	@docker ps -a | grep noobaa_ | awk '{print $$NF}' | xargs -r docker stop
+	@docker ps -a | grep noobaa_ | awk '{print $$NF}' | xargs -r docker rm
```

Comment:
The -r option will work at Linux but will not work at macOS

```
MacBook-Pro$ xargs -r
xargs: illegal option -- r
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
```
we can use:
```
@docker ps -a | grep noobaa_ | awk '{print $$NF}' | xargs docker stop &> /dev/null
@docker ps -a | grep noobaa_ | awk '{print $$NF}' | xargs docker rm &> /dev/null
```

Reply:
another issue here is that on the original line we use `awk '{print $$NF}'` which looks like not working on macOS as well, so we can replace it with `awk '{print $1}'`
so the change can be:
```
@docker ps -a | grep noobaa_ | awk '{print $1}' | xargs docker stop &> /dev/null
@docker ps -a | grep noobaa_ | awk '{print $1}' | xargs docker rm &> /dev/null
```

Reply:
That's certainly a challenge - I don't have a Mac to test on, and I don't believe the CI has it either.

Should the pipeline be >& and not &>  to /dev/null ?

Reply:
for bash and zsh >& and &> (and also 2>&1 ) redirect stderr to stdout
it will make a difference if we want to use (support other shells such as csh, tcsh or ksh)

Writing &> will work on bash and zsh
Writing >& will work on bash, zsh, csh and tcsh (not ksh)